packages/formstr-app/src/containers/CreateFormNew/components/FormSettings/index.tsx

```diff
@@ -87,6 +101,40 @@ function FormSettings() {
         </Panel>
 
         <Panel header="Customization" key="customization">
+          <div className="property-setting">
```

Comment:
We should probably have a seperate component for customization settings but this is alright